visidata/sheets.py

```diff
@@ -963,7 +963,7 @@ def getSheet(self, k):
 
     def addRow(self, sheet, **kwargs):
         super().addRow(sheet, **kwargs)
-        if not self.options.load_lazy:
+        if not self.options.load_lazy and not sheet.options.load_lazy:
```

Comment:
`and` was chosen to fit @geekscrapy's particular use-case here: https://github.com/saulpw/visidata/issues/1189

`options.load_lazy` would be `False` by default, and sub-sheets that geekscrapy did not want loaded would be set to `True`.